src/test/java/net/datafaker/providers/entertainment/StarWarsTest.java

```diff
@@ -2,48 +2,35 @@
 
 import org.junit.jupiter.api.Test;
 
+import java.util.Arrays;
+import java.util.Collection;
+
 import static org.assertj.core.api.Assertions.assertThat;
 
 
 class StarWarsTest extends EntertainmentFakerTest {
 
-    @Test
-    void character() {
-        assertThat(faker.starWars().character()).isNotEmpty();
-    }
-
     @Test
     void callSign() {
         assertThat(faker.starWars().callSign()).isNotEmpty().matches("\\w+\\s(Leader|\\d)$");
     }
 
     @Test
-    void species() {
-        assertThat(faker.starWars().species()).isNotEmpty();
-    }
-
-    @Test
-    void planets() {
-        assertThat(faker.starWars().planets()).isNotEmpty();
-    }
-
-    @Test
-    void droid() {
-        assertThat(faker.starWars().droids()).isNotEmpty();
-    }
-
-    @Test
-    void wookieWords() {
-        assertThat(faker.starWars().wookieWords()).isNotEmpty();
+    void alternativeSpelling() {
```

Comment:
This can’t be done the new way? (I’m on my
Phone and couldn’t check the code or data file

Reply:
There are some methods in different classes including this one which are a bit trickier under the hood...
e.g. this method looks like 
```java
    public String alternateCharacterSpelling() {
        return resolve("star_wars.alternate_character_spellings." + CHARACTERS[faker.random().nextInt(CHARACTERS.length)]);
    }
```
so it requires some more sophisticated handling... 
I would rather migrate everything that's possible to new way and then try to handle the rest